osmaxx/conversion/converters/utils.py

```diff
@@ -3,6 +3,15 @@
 
 import os
 
+# Use the built-in version of scandir if possible, otherwise
+# use the scandir module version
+try:
+    from os import scandir
+except ImportError:
+    from scandir import scandir
+except:
+    pass
```

Comment:
Don't we want to fail here (rather than later at runtime) when both exports were unsuccessful?